.jenkins/Jenkinsfile

```diff
@@ -184,11 +184,10 @@ def win2016LinuxElfBuild(String version, String compiler, String build_type) {
                 checkout scm
                 unstash "linux-${compiler}-${build_type}-${version}-${BUILD_NUMBER}"
                 bat 'move build linuxbin'
-                powershell 'Copy-Item -Recurse C:\\openenclave\\prereqs\\nuget ${env:WORKSPACE}\\prereqs'
                 dir('build') {
                   bat """
                       vcvars64.bat x64 && \
-                      cmake.exe ${WORKSPACE} -G Ninja -DADD_WINDOWS_ENCLAVE_TESTS=ON -DBUILD_ENCLAVES=OFF -DUSE_LIBSGX=ON -DCMAKE_BUILD_TYPE=${build_type} -DLINUX_BIN_DIR=${WORKSPACE}\\linuxbin\\tests -Wdev && \
+                      cmake.exe ${WORKSPACE} -G Ninja -DADD_WINDOWS_ENCLAVE_TESTS=ON -DBUILD_ENCLAVES=OFF -DUSE_LIBSGX=ON -DCMAKE_BUILD_TYPE=${build_type} -DLINUX_BIN_DIR=${WORKSPACE}\\linuxbin\\tests -DNUGET_PACKAGE_PATH=C:/openenclave/prereqs/nuget -Wdev && \
```

Comment:
I think it's generally better to not use a hardcoded path.

While this is a Jenkins file, it might be good to get in the habit of doing something such as:
```
DNUGET_PACKAGE_PATH=${OE_DIR}/prereqs/nuget```

Reply:
Yes, this is a good idea. We should parameterize this if possible, and that way someone submitting a sandbox run can set this OE_DIR variable to be whatever they want. I'll create an issue to track this (as it was previously hardcoded as well, and I think can be addressed outside the scope of this PR)